third_party/chromium/media/base/starboard_utils.cc

```diff
@@ -51,6 +51,9 @@ int GetBitsPerPixel(const std::string& mime_type) {
   return 8;
 }
 
+const char* kVp9_8k_mime = "video/webm; codecs=\"vp9\"; width=7680; height=4320";
```

Comment:
nit: constexpr, and move them before the function.

Reply:
removed.